src/Field/Definition/BloksField.php

```diff
@@ -109,7 +110,7 @@ public function validateData (ComponentContext $context, array $contentPath, mix
 				$component = $context->getComponentByKey($componentData["component"]);
 				$component->validateData(
 					$context,
-					$data,
+					$componentData,
```

Comment:
This fixes a bug that I just encountered where we were always validating the entire API data dump instead of just the nested component data, which will ultimately lead to validation errors since the nested components can't find their associated field values:

```php
$allData = [
	0 => [ … ],
	1 => [ … ],
];

// vs.

$nestedData = [ … ];
```

Reply:
Nice catch 👍